src/app/garfile/departure/post.controller.js

```diff
@@ -85,7 +85,8 @@ const performAPICall = (cookie, res, buttonClicked) => {
       if (buttonClicked === 'Save and continue') {
         res.redirect('/garfile/arrival');
       } else {
-        res.redirect('/home');
```

Comment:
This is almost a hack of sorts.

If "Save and Exit" is pressed, we essentially want to redirect to the /garfile/view page, but the problem is a normal redirect will perform a GET, which doesn't work. It only accepts POST calls.

307 essentially translate to "POST again" with the same details, which works with the current /garfile/view and thus loads back the GAR for the summary display.

Would have been much simpler to have been a GET call or something, but essentially, any redirects previously to '/home' will need to be 307, '/garfile/view'